roadmap:
  # === High level settings === #
  debug_output: false   # Print debug information
  update_frequency: 10  # (Hz) Publish map information at this rate
  map_package_name: 'roadmap'         # Package where the map is
  map_file_name: 'maps/loop.xml'  # Name of the map file
  # map_file_name: 'maps/test_map.xml'  # Name of the map file  
  # map_file_name: "maps/test_map.yaml"
  # map_file_name: 'maps/test_map.osm'

  # === Spline settings === #
  spline:
    fit_clothoid: true             # Fits a clothoid before fitting cubic splines if TRUE
    minimum_waypoint_distance: 2.0  # Ignore waypoints closer than this distance together
    spline_sample_distance: 1       # Distance between sampled waypoints on the spline
    clothoid_point_per_xm: 1        # Distance forced between points on the Clothoid

  # === Visual settings === #
  scale: 0.3                       # Scale of the visuals
  
  # === Topics === #
  # Topic to read waypoints from INSTEAD of using a predefined map
  external_waypoint_topic: '' #/carla/ego_vehicle/waypoints' 